packages/x-components/tests/e2e/cucumber/facets/facets.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+Given('no special config for layout view', () => {
+  cy.visit('/?useMockedAdapter=true');
+});
+
+Then(
+  'clicked filter number {int} is shown in the selected filters list is {boolean}',
```

Comment:
There are situations where a filter is selected but not because it was clicked and the way of testing it would be the same. If it's not necessary, get rid of the `clicked`.